python/ManagingMultipleLargeScaleJobs/ManagingMultipleLargeScaleJobs.ipynb

```diff
@@ -233,16 +233,33 @@
     "    spatial_extent = row[\"spatial_extent\"]\n",
     "\n",
     "    # Build the openEO process\n",
-    "    s2_cube = connection.load_collection(\n",
+    "    scl = connection.load_collection(\n",
     "        collection_id=\"SENTINEL2_L2A\",\n",
     "        spatial_extent=spatial_extent,\n",
     "        temporal_extent=[\"2022-06-01\",\"2022-09-01\"],\n",
-    "        bands=[\"B02\", \"B03\", \"B04\", \"SCL\"],\n",
+    "        bands=[\"SCL\"],\n",
     "        properties={\"eo:cloud_cover\": lambda x: x.lte(60)}\n",
     "    )\n",
     "\n",
+    "    cloud_mask = scl.process(\n",
+    "        \"to_scl_dilation_mask\",\n",
+    "        data=scl,\n",
+    "        kernel1_size=17, kernel2_size=77,\n",
+    "        mask1_values=[2, 4, 5, 6, 7],\n",
+    "        mask2_values=[3, 8, 9, 10, 11],\n",
+    "        erosion_kernel_size=3)\n",
+    "\n",
+    "    s2_cube = connection.load_collection(\n",
+    "        collection_id=\"SENTINEL2_L2A\",\n",
+    "        spatial_extent=spatial_extent,\n",
+    "        temporal_extent=[\"2022-06-01\",\"2022-09-01\"],\n",
+    "        bands=[\"B02\", \"B03\", \"B04\"],\n",
+    "        properties={\"eo:cloud_cover\": lambda x: x.lte(60)}\n",
+    "    )\n",
     "    # Mask out clouds\n",
-    "    s2_cube = s2_cube.process(\"mask_scl_dilation\", data=s2_cube, scl_band_name=\"SCL\").filter_bands(s2_cube.metadata.band_names[:-1])\n",
+    "    s2_cube = s2_cube.mask(cloud_mask)\n",
+    "\n",
+    "    s2_cube = s2_cube.filter_bands(s2_cube.metadata.band_names[:-1])\n",
```

Comment:
What happened here is that we took out the SCL band from the cube, as it is the last entry in band names. This should no longer be needed.